README.md

```diff
@@ -1,6 +1,6 @@
 # Node.js
 
-Node.js is an open-source, cross-platform, JavaScript runtime environment.
+Node.js is an open-source, cross-platform JavaScript runtime environment.
```

Comment:
I kind of wonder if we eventually want to do something more like this:

```markdown
Node.js is a JavaScript runtime environment. It allows you to run JavaScript
without a web browser. Node.js is open source and cross-platform.
```

But...not in this PR.  :-D